src/algorithms/math/fibonacci.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Show fibonacci sequence up to a given number
+ * @param {Number} num given number
+ */
+const fibonacci = (num) => {
```

Comment:
Wouldn't it be better if it was the `nth` number instead of up until a certain number value?